ui/src/app/settings/components/project-details/project-details.tsx

```diff
@@ -573,6 +574,30 @@ export class ProjectDetails extends React.Component<RouteComponentProps<{name: s
                     </div>
                 )}
 
+                <h4>Whitelisted namespaced resources {helpTip('Namespace-scoped K8s API Groups and Kinds which are permitted from being deployed')}</h4>
```

Comment:
Please update as following for consistency:

```suggestion
                <h4>Whitelisted namespaced resources {helpTip('Namespace-scoped K8s API Groups and Kinds which are permitted to deploy')}</h4>
```